exchanges/bithumb/bithumb_test.go

```diff
@@ -92,7 +92,7 @@ func TestGetTicker(t *testing.T) {
 // not all currencies have dates and fluctuation rates
 func TestGetAllTickers(t *testing.T) {
 	t.Parallel()
-	tick, err := b.GetAllTickers(context.Background())
+	tick, err := b.GetAllTickers(t.Context())
 	require.NoError(t, err, "GetAllTickers must not error")
 	for _, res := range tick {
 		assert.Positive(t, res.OpeningPrice, "OpeningPrice should be positive")
```

Comment:
[This is causing build issues](https://github.com/thrasher-corp/gocryptotrader/actions/runs/14749428048/job/41403183465?pr=1893#step:11:329) because we foolishly assume that an exchange should always have trades happening or a currency is still being used.

Not your fault but I'd double-approve if its just changed that `tick` is not empty